GeoTIFF_Standard/standard/annex-b.adoc

```diff
@@ -89,6 +89,13 @@ All key-values which are not of type SHORT are to be stored in one of the follow
 
 This tag is used to store all of the DOUBLE valued GeoKeys, referenced by the GeoKeyDirectoryTag. The meaning of any value of this double array is determined from the GeoKeyDirectoryTag reference pointing to it. FLOAT values should first be converted to DOUBLE and stored here.
 
+GeoLongParamsTag:
+    Tag = 34738 (87B2.H)
+    Type = LONG (int32 or uint32)
+    N = variable
+
+This tag is used to store all of the LONG oand ULONG valued GeoKeys, referenced by the GeoKeyDirectoryTag. The meaning of any value of this long array is determined from the GeoKeyDirectoryTag reference pointing to it. The interpretation fot he bytes as LONG or ULONG depen on the datatype indicated in the GeoKeyDirectoryTag.

```

Comment:
as it is a TIFF tag, we must comply with the specification and if we declare it to be of type LONG it can only contain such value (ant not SLONG, signed int32, values)